handlers/main.yml

```diff
@@ -56,3 +39,17 @@
 - name: update running audit failure mode
   command: auditctl -f {{ rhel7stig_auditd_failure_flag }}
   failed_when: no
+
+- name: init aide
+  command: /usr/sbin/aide --init -B 'database_out=file:{{ rhel7stig_aide_temp_db_file }}'
+  async: "{{ rhel7stig_aide_init_timeout | default(omit) }}"
+  poll: "{{ rhel7stig_aide_init_interval | default(omit) }}"
+  when: not rhel7stig_aide_db_status.stat.exists
+  notify: move aide db
+  tags:
```

Comment:
Is there a need for tags in the handlers file?

Reply:
There used to be a need but it looks like they fixed that.